library/CM/Maintenance/RunEventJob.php

```
@@ -21,7 +21,13 @@ protected function _execute(CM_Params $params) {
             throw $e;
         } finally {
             $newRelic->endTransaction();
-            $maintenance->handleClockworkEventResult($eventName, $result);
+            try {
+                $maintenance->handleClockworkEventResult($eventName, $result);
+            } catch (CM_Exception_Invalid $e) {
```

As discussed let's remove this empty catch. This situation (even name is unknown) can be fatal (or not), but we cannot know it - so better throw fatal.